dangerfile.ts

```diff
@@ -1,6 +1,8 @@
 import * as child from 'child_process'
 
-import { danger, fail, warn } from 'danger'
+import { danger, fail, schedule, warn } from 'danger'
+
+const shouldRun = !danger.github || (danger.github && danger.github.pr.user.type !== 'Bot');
```

Comment:
Add this up here as we're checking if we should run asynchronously now as well as functionally.